packages/sanity/src/desk/components/paneHeaderActions/InsufficientPermissionsMessageTooltip.tsx

```diff
@@ -22,7 +22,15 @@ export function InsufficientPermissionsMessageTooltip({
 
   return (
     <Tooltip
-      content={loading ? 'Loading…' : <InsufficientPermissionsMessage currentUser={currentUser} />}
+      content={
+        loading ? (
```

Comment:
Just noticed that tooltip for the create-document button in document list is unstyled when loading, this improves it.